frontend/src/metabase/query_builder/containers/ArchiveQuestionModal.jsx

```diff
@@ -34,10 +34,22 @@ class ArchiveQuestionModal extends Component {
       ? t`This model will be removed from any dashboards or pulses using it.`
       : t`This question will be removed from any dashboards or pulses using it.`;
 
+    const widgetCount = question.getDropdownWidgetCountForValues();
```

Comment:
The name seems off. Should we change it something neutral? I really like the suggestion in comments about `usage_count`.

Reply:
I'm not sure I've seen that comment. What do you recommend? `getWidgetUsageCount()`? Something else?

Reply:
`parameter_usage_count` maybe? We count how many times this card was used in a parameter, right?